docs/canonicalk8s/charm/howto/ceph-csi.md

```diff
@@ -165,6 +169,10 @@ juju deploy -n 1 ceph-osd-alt ceph-osd \
 juju deploy ceph-csi-alt ceph-csi \
     --config create-namespace=true \
     --config namespace=${CEPH_NS_ALT} \
+    --config ceph-xfs-storage-class-name-formatter="ceph-xfs-{app}" \
+    --config ceph-ext4-storage-class-name-formatter="ceph-ext4-{app}" \
+    --config cephfs-storage-class-name-formatter="cephfs-{name}-{app}" \
```

Comment:
I think including the {name} and {app} here could be a tad confusing because we don't explicitly call it out. Users may believe they can copy and paste the entire code block as they were able to in the previous block. We set the namespace alt be default.